src/universal/components/Badge/Badge.js

```diff
@@ -31,11 +39,12 @@ Badge.defaultProps = {
   value: 0
 };
 
+const boxShadow = '1px 1px 2px rgba(0, 0, 0, .5)';
```

Comment:
Yeah, I agree and will move them into `ui.js` as the patterns normalize. Some are already there.